### DISCUSSION TITLE
Retrieving the Currently "Connecting" Connector in Wagmi V2

### DISCUSSION THREAD

**Discussion by frankfka** [wevm/wagmi#3411] [Q&A]
Is it possible to retrieve the currently active connector when in a `connecting` state? I would expect it to come from `connector` in `useAccount` but that doesn't seem to be the case

**Comment by sameoldlab** [comment]
`useAccount` only returns a connector if `status === 'connected'`. But the `connect` and `reconnect` actions take in a connector or connector array as parameters so you could save the value before calling them. The actions also update `config` state as which `connector.connect` doesn't seem to do.

**Reply by frankfka** [reply]
We can store state locally, but it'd be nice to have `wagmi` expose this instead
